contributing-docs/03_contributors_quick_start.rst

```diff
@@ -601,7 +601,7 @@ All Tests are inside ./tests directory.
 
    root@63528318c8b1:/opt/airflow# pytest tests/utils/test_dates.py
    ============================================================= test session starts ==============================================================
-   platform linux -- Python 3.8.16, pytest-7.2.1, pluggy-1.0.0 -- /usr/local/bin/python
+   platform linux -- Python 3.9.16, pytest-7.2.1, pluggy-1.0.0 -- /usr/local/bin/python
```

Comment:
FWIW 😅

```suggestion
   platform linux -- Python 3.9.20, pytest-8.3.3, pluggy-1.5.0 -- /usr/local/bin/python
```

You could change the other references to 3.9 subversions in the other files if you want, I assume that it's not too critical

Reply:
Updated the full pytest logs :-D